src/libsnark_helpers/libsnark_helpers.tcc

```diff
@@ -200,17 +89,17 @@ void write_setup(libsnark::r1cs_ppzksnark_keypair<ppT> keypair, boost::filesyste
 	boost::filesystem::path path_vk_raw = setup_dir / vk_raw;
 	boost::filesystem::path path_pk_raw = setup_dir / pk_raw;
 
-    libsnark::r1cs_ppzksnark_proving_key<ppT> proving_key = keypair.pk;
-    libsnark::r1cs_ppzksnark_verification_key<ppT> verification_key = keypair.vk;
+    provingKeyT<ppT> proving_key = keypair.pk;
+    verificationKeyT<ppT> verification_key = keypair.vk;
 
-	verificationKey_to_json<ppT>(verification_key, path_vk_json);
+	verificationKeyToJson<ppT>(verification_key, path_vk_json);
 
 	serializeVerificationKeyToFile<ppT>(verification_key, path_vk_raw);
 	serializeProvingKeyToFile<ppT>(proving_key, path_pk_raw);
 };
 
 template<typename ppT>
-void r1cs_constraints_to_json(libsnark::linear_combination<libff::Fr<ppT> > constraints, boost::filesystem::path path)
+void r1csConstraintsToJson(libsnark::linear_combination<libff::Fr<ppT> > constraints, boost::filesystem::path path)
```

Comment:
You switched from snake case to camel case here, but I can see some functions in the cpp code that remain snake_case. If you prefer camelCase I'm happy with that, but we might want to harmonize it to all the code to keep it consistent. (Note that I tried to use snake_case most of the time (I guess there were some irregularities) to stick to the convention used in libsnark)

Reply:
I modified them in order to have the same style as the functions from lines 30-49 https://github.com/clearmatics/zeth/blob/develop/src/libsnark_helpers/libsnark_helpers.hpp. Probably I had to modify all of them in snake case. 
Did you use different styles for purpose?